ui/src/message_popup/mobile_prototype_card.jsx

```diff
@@ -0,0 +1,239 @@
+      academicPerformance,
+      interests,
+      familyBackground,
+      ses
+    } = this.state.question.student;
+      
+    const studentDialogActions = [
+      <FlatButton
+        label="Close"
+        primary={true}
+        onTouchTap={this.closeStudentDialog}
+      />
+      
+    ];
+    
+    return (
+      <div>
+        <div>
+          <AppBar 
+            title='Message PopUp Practice'/>
+        </div>
+        <div style={styles.questionContainer}>
+          {<Message text={this.state.question.text} question={this.state.question} type='student' openStudentDialog={this.openStudentDialog} />}
+          {_.map(this.state.responses, (response) => {
+            return (
+              <Message text={response} question={this.state.question} type='user' />
+            );
+          })}
+        </div>
+        <div style={styles.responseContainer}>
+          <Divider />
+          <div style={styles.responseTextBox}>
+            <TextField
+              id="responseTextField"
+              multiLine={true}
+              hintText="Type something you would say"
+              textareaStyle={styles.responseTextAreaInner}
+              rows={2}
+              underlineShow={false}
+              rowsMax={2}
+              disabled={this.state.responses.length > 10}
+              value={this.state.currentResponse}
+              onChange={this.onTextChange}
+              style={{width: '100%', paddingLeft: 20, paddingRight: 20}}
+            />
+          </div>
+          <div style={styles.responseButtonRow}>
+            <RaisedButton 
+              label={this.state.responses.length > 10 ? 'Next' : 'Send'}
+              secondary={true}
+              onTouchTap={this.state.responses.length > 10 ? this.next : this.sendMessage}
+            />
+          </div>
+        </div>
+        
+        <Dialog
+          title="Student Information"
+          actions={studentDialogActions}
+          modal={false}
+          open={this.state.currentStudent !== undefined}
+          style={{backgroundColor: 'rgba(0, 0, 0, 0.1)'}}
+          onRequestClose={this.closeStudentDialog}
+        >
+          <div style={styles.studentName}>{name}</div>
+          <div style={styles.studentAttribute}>{`${grade} ${gender}, ${race}`}</div>
+          {behavior && <div style={styles.studentAttribute}>{behavior}</div>}
+          {ell && <div style={styles.studentAttribute}>{ell}</div>}
+          {learningDisabilities && <div style={styles.studentAttribute}>{learningDisabilities}</div>}
+          {academicPerformance && <div style={styles.studentAttribute}>{academicPerformance}</div>}
+          {interests && <div style={styles.studentAttribute}>{interests}</div>}
+          {familyBackground && <div style={styles.studentAttribute}>{familyBackground}</div>}
+          {ses && <div style={styles.studentAttribute}>{ses}</div>}
+        </Dialog>
+        
+      </div>
+    );
+  },
+});
+
+const Message = React.createClass({
```

Comment:
Could these be split into different files?  You can also make a separate folder if there's more files and it's helpful for organizing, up to you.